### ISSUE TITLE
forge: show current variable values during debug

### ISSUE THREAD

**Issue by crisgarner** [gakonst/foundry#410]
Add some way to show the current value of the variables while debugging

**Comment by gakonst** [comment]
It would be nice if you could be more specific, do you mean like Watch expressions?

**Comment by onbjerg** [comment]
I think the ask is to show the values of variables in the current scope, similar to how Truffle's debugger can output variables in scope (their `v` command).

**Comment by crisgarner** [comment]
> I think the ask is to show the values of variables in the current scope, similar to how Truffle's debugger can output variables in scope (their `v` command).

Exactly, just like truffle debugger. Here is a screenshot from the visual debugger: 

![image](https://user-images.githubusercontent.com/578688/149048736-c1c855fe-c20b-4ab4-886f-898a71be6a87.png)

https://github.com/trufflesuite/truffle-plugin-debugger

**Comment by mattsse** [comment]
The debugger now shows the values of variables in the current scope, added in #15141 with follow-up fixes in #15200. Further debugger improvements are tracked in #927.
